packages/fiori/src/i18n/messagebundle.properties

```diff
@@ -173,7 +173,7 @@ WIZARD_NAV_ARIA_ROLE_DESCRIPTION = Wizard
 WIZARD_NAV_STEP_DEFAULT_HEADING=Step
 
 #XTIT: View Settings Dialog dialog title for Sort
```

Comment:
Why do we change the title?

Reply:
Because previously it was with wrong text